gradio/components.py

```diff
@@ -10,13 +10,12 @@
 import numbers
```

Comment:
nit-pick: Let's delete `save_flagged`, `restore_flagged` from the Carousel as those methods are deprecated?